src/libraries/System.Diagnostics.EventLog/src/System/Diagnostics/EventLogInternal.cs

```diff
@@ -1046,8 +1046,7 @@ public void ModifyOverflowPolicy(OverflowAction action, int retentionDays)
 
         private void OpenForRead(string currentMachineName)
         {
-            if (this.boolFlags[Flag_disposed])
-                throw new ObjectDisposedException(GetType().Name);
+            ObjectDisposedException.ThrowIf(this.boolFlags[Flag_disposed], this);
```

Comment:
Does this build?  I thought this library still built against things other than netcoreapp latest.